talk/objectorientation.tex

```diff
@@ -634,7 +634,8 @@
     \end{itemize}
   \end{block}
   \begin{exampleblock}{Practically}
-    \begin{cppcode}
+   \begin{minipage}{0.7\textwidth}
+    \begin{cppcode*}{highlightlines={2,7}}
```

Comment:
Does this compile ? Where is the \end{minipage} statement ?

Reply:
```suggestion
    \begin{cppcode}
```

Reply:
> Does this compile ? Where is the \end{minipage} statement ?

It's after the code block, but it was only needed to limit the width of the code block:
https://github.com/hageboeck/cpluspluscourse/blob/8068490708f516569609ab8f7f7d9408e89a7fb7/talk/objectorientation.tex#L650

If no lines are highlighted, you don't need the minipage.

I think I'll take it out.

Reply:
Ouch, it's in another commit. I'll fix it.